pkg/rancher-desktop/preload/extensions.ts

```diff
@@ -266,18 +266,18 @@ class Client implements v1.DockerDesktopClient {
     openExternal: (url: string) => {
       ipcRenderer.send('extensions/open-external', url);
     },
-    platform: '<unknown>',
-    arch:     '<unknown>',
+    platform: process.platform,
+    arch:     process.arch,
```

Comment:
So `host.arch` defaults to `process.arch`, but can be overridden by passing a value into the constructor. I can see the default behavior causing subtle bugs - maybe it would be better to require `arch` to be passed in the constructor?  I see that we always set it explicitly anyways.